annotation_main:
  visualize_service_name: visualize_cloud_service
  sensor_service_name: sensor_service
  mesh_service_name: mesh_service
  hdf5_record_service_name: record_acc_service
  tf_br_service_name: tf_broadcast_service
  nearest_radious: 0.002
  LEAF_SIZE: 0.001
  radious: 0.05
  xyz_step: 0.003
  qxyz_step: 0.01
  main_object_name: HV8
  the_number_of_dataset: 1
  
decide_object_position:
  sensor_name: phoxi_camera
  sensor_angle_max: 0.087
  sensor_angle_min: 0.087
  sensor_distance_min: 0.8
  sensor_distance_max: 1.5
  z_position: 0.003
  box_height: 0.01
  box_name: denso_box
  object_height: 0.15
  radious: 0.05

data_transform_pkg:
  f_scale: 0.5
  cx_scale: 0.99
  cy_scale: 0.9

data_3D_to_2D:
  radious_x: 0.05
  radious_y: 0.05